models/brain_image_synthesis_latent_diffusion_model/docs/README.md

```diff
@@ -0,0 +1,75 @@
+# Brain Imaging Generation with Latent Diffusion Models
+
+### **Authors**
+
+Walter H. L. Pinaya, Petru-Daniel Tudosiu, Jessica Dafflon, Pedro F Da Costa, Virginia Fernandez, Parashkev Nachev,
+Sebastien Ourselin, and M. Jorge Cardoso
+
+### **Tags**
+Synthetic data, Latent Diffusion Model, Generative model, Brain Imaging
+
+## **Model Description**
+This model is trained using the Latent Diffusion Model architecture [1] and is used for the synthesis of conditioned 3D
+brain MRI data. The model is divided into two parts: an autoencoder with a KL-regularisation model that compresses data
+into a latent space and a diffusion model that learns to generate conditioned synthetic latent representations. This
+model is conditioned on age, sex, the volume of ventricular cerebrospinal fluid, and brain volume normalised for head size.
+
+![](./figure_1.png) <br>
+<p align="center">
+Figure 1 - Synthetic image from the model. </p>
+
+
+## **Data**
+The model was trained on brain data from 31,740 participants from the UK Biobank [2]. We used high-resolution 3D T1w MRI with voxel size of 1mm3, resulting in volumes with 160 x 224 x 160 voxels
+
+#### **Preprocessing**
+We used UniRes [3] to perform a rigid body registration to a common MNI space for image pre-processing. The voxel intensity was normalised to be between [0, 1].
+
+## **Performance**
+This model achieves the following results on UK Biobank: an FID of 0.0076, an MS-SSIM of 0.6555, and a 4-G-R-SSIM of 0.3883.
+
+Please, check Table 1 of the original paper for more details regarding evaluation results.
+
+
+## **commands example**
+
+Execute sampling:
+
+```shell
+export PYTHONPATH=$PYTHONPATH:"<path to 'GenerativeModels'>"
```

Comment:
what path should be added to the path? Seems all components have been merged into the core?

Reply:
Yes, it was well observed. This is not necessary anymore.